optimize/util/optimize-commons/src/main/resources/service-config.yaml

```diff
@@ -386,7 +386,7 @@ opensearch:
     index:
       # the prefix prepended to all Optimize index and alias names
       # NOTE: Changing this after Optimize was already run before, will create new empty indexes
-      prefix: ${CAMUNDA_OPTIMIZE_OPENSEARCH_SETTINGS_INDEX_PREFIX:optimize}
+      prefix: ${CAMUNDA_OPTIMIZE_OPENSEARCH_SETTINGS_INDEX_PREFIX:}
```

Comment:
I think this gets parsed to null if I'm not mistaken, so an isEmpty call would NPE

Reply:
Good point! Let me check, I assumed that with the colon it would interpret it as ""

Reply:
I added a test in `ConfigurationServiceTest` and it get parsed as ""